unit-tests/port/test-port-legacy.py

```diff
@@ -0,0 +1,100 @@
+# License: Apache 2.0. See LICENSE file in root directory.
```

Comment:
I suggest not to have a single file with all the test.
Lets make a test file for every test group.

This way if some tests will be unstable it will state the name and you will know exactly which one without investigating each time

Reply:
changed